projects/packages/changelogger/src/Plugins/WordpressVersioning.php

```diff
@@ -151,22 +151,32 @@ public function nextVersion( $version, array $changes, array $extra = array() )
 	}
 
 	/**
-	 * Extract the index and count from a prerelease string.
+	 * Extract the index and values from a prerelease string.
 	 *
 	 * @param string|null $s String.
-	 * @return array Two elements: index and count.
+	 * @return array Two elements: the index value of the pattern matched, and an array of matched capture group values.
 	 * @throws InvalidArgumentException If the string is invalid.
 	 */
 	private function parsePrerelease( $s ) {
 		if ( null === $s ) {
-			return array( 100, 0 );
+			return array( 100, array( 0 => 0 ) );
 		}
-		foreach ( array( 'dev', 'alpha(\d*)', 'beta(\d*)', 'rc(\d*)' ) as $i => $re ) {
+
+		$arr = array();
+		foreach ( array( 'dev', 'alpha(\d*)', '(\d\d(?:0[1-9]|1[0-2]))\.(\d+)', 'beta(\d*)', 'rc(\d*)' ) as $i => $re ) {
 			if ( preg_match( "/^{$re}\$/", $s, $m ) ) {
-				return array( $i, isset( $m[1] ) ? (int) $m[1] : 0 );
+				$arr[] = $i;
+				$arr[] = isset( $m[1] ) ? array_slice( $m, 1 ) : array( 0 => 0 );
+
+				break;
 			}
 		}
-		throw new InvalidArgumentException( "Invalid prerelease string \"$s\"" ); // @codeCoverageIgnore
+
+		if ( empty( $arr ) ) {
```

Comment:
`$arr` will always be set, no need to use `empty()` here.
(unneeded use of `empty()` is one of my pet peeves)

```suggestion
		if ( ! $arr ) {
```

Reply:
TIL, I'm too used to JS I guess :)